behind-the-veil-siteroot/imports/api/publications/reviews.js

```diff
@@ -16,3 +16,15 @@ Meteor.publish('specific_review', function(bookingId) {
     // Check if the username matches either the user
     return ReviewCollection.find({_id: bookingId});
 });
+
+/**
+ * Publishes all reviews to the client.
+ * This method does not apply any filtering and will publish all documents
+ * in the ReviewCollection. Use this method cautiously, especially if the
+ * collection is large, as it may impact performance.
+ *
+ * @returns {Mongo.Cursor} - A cursor representing the result set of all reviews to be published.
+ */
+Meteor.publish('all_reviews', function() {
```

Comment:
Should this be getting all reviews ever to exist or just all reviews specific to that user?